pkg/scheduler/algorithm/priorities/balanced_resource_allocation.go

```diff
@@ -41,10 +41,16 @@ var (
 func balancedResourceScorer(requested, allocable *schedulernodeinfo.Resource, includeVolumes bool, requestedVolumes int, allocatableVolumes int) int64 {
 	cpuFraction := fractionOfCapacity(requested.MilliCPU, allocable.MilliCPU)
 	memoryFraction := fractionOfCapacity(requested.Memory, allocable.Memory)
+
+	if cpuFraction >= 1 || memoryFraction >= 1 {
```

Comment:
This has already been addressed in https://github.com/kubernetes/kubernetes/pull/79453

Reply:
/close #79453 has already get merged.

Thank you anyway for open this PR, @prtsh

Reply:
Thanks for the review @draveness